feature-libs/product-configurator/rulebased/components/add-to-cart-button/configurator-add-to-cart-button.component.ts

```diff
@@ -223,4 +230,34 @@ export class ConfiguratorAddToCartButtonComponent {
         }
       });
   }
+  leaveConfigurationOverview(): void {
+    this.container$.pipe(take(1)).subscribe((container) => {
+      if (
+        container.routerData.owner.type ===
+        CommonConfigurator.OwnerType.ORDER_ENTRY
+      ) {
+        this.goToOrderDetails(container.routerData.owner);
+      } else if (
```

Comment:
do we need this if statement? Because we can only have 2 cases here. DisplayOnly is only called for order history or review order

Reply:
Will we only have 2 cases? If yes, that's a valid point. But what about other processes where you can view the configuration as well without being able to change it (e.g. Return Process)? Otherwise, I think we should keep the method this way.

Reply:
We can never change the configuration for an order, therefore I don't understand your comment...
From my point of view: We only have those 2 cases

Reply:
Ok, done.